src/components/Login/Login.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import { Grid, Paper, Avatar } from "@mui/material";
+import { TextField, Button, Typography, Link } from "@mui/material";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+
+import "./login.css";
+
+const Login = () => {
+  return (
+    <Grid>
```

Comment:
What purpose is this Grid serving? Seems like an extra node. Try removing it to see what it does and if it does nothing, remove it.